app/views/hearings/show.html.haml

```diff
@@ -0,0 +1,75 @@
+= govuk_page_title(@hearing_day.strftime('%d/%m/%Y'), t('generic.hearing'))
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = 'Hearing'
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Hearing type
+      %td.govuk-table__cell
+        = @hearing.hearing_type
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Court
+      %td.govuk-table__cell
+        = @hearing.court_name || 'not set'
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Time listed
+      %td.govuk-table__cell
+        = @hearing_day&.strftime('%H:%M')
+
+%br
+.govuk-heading-m
+  = 'Attendees'
```

Comment:
- We could remove `%br` as the preceding table element will provide relevant spacing.
- we should add the relevant typography tag to make `.govuk-heading-m` semantically correct
- add `attendees` string to translation

@jsugarman

Reply:
@naseberry not sure what you mean by point 2, example, but others done

Reply:
point 2, should it be a heading? Maybe an h2

Reply:
done now